mypy.ini

```diff
@@ -1,5 +1,35 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
```

Comment:
I really feel like this should be unnecessary. I've never had problems with it. What problem does it address?

Reply:
I'll try to give you concrete examples when I have time.

But off the top of my head, it can lead to issues with mypy picking up python files copied or created in these folders.
For example mypy might think there's a duplicate module and not able to differentiate it. Or give errors for python modules not part of the source code. Even when it doesn't error it'll speed up the run by not scanning useless files.

I don't remember if you can tell mypy to follow a `.gitignore`, but that's irrelevant given you prefer not using one.

I know the skeleton uses tox. I don't remember if it has anything that should generate a "build" folder.

Reply:
I've decided to remove this for now. Until there's a good justification for it, I'd rather it not be here.